app.yaml

```diff
@@ -40,9 +40,17 @@ handlers:
     # WARNING TO DEVELOPERS: Files in this folder may be stale for
     # up to 10 mins after a code release.
     Cache-Control: 'public, max-age=600'
+- url: /assets/(constants\.json)
```

Comment:
Ah actually -- having it as a JS constant may not work for interpolating the script src URLs. @BenHenning suggests just running a sed command on each release to replace ASSET_DIR_PREFIX with the build slug for that release, but we'd still need to figure out what to do in develop -- it's not clear at this point.

I think this can still be left for a separate issue/PR though.